src/activity_actor.cpp

```diff
@@ -2546,7 +2546,7 @@ void lockpick_activity_actor::finish( player_activity &act, Character &who )
     } else if( furn_type == f_gunsafe_ml && lock_roll > ( 3 * pick_roll ) ) {
         who.add_msg_if_player( m_bad, _( "Your clumsy attempt jams the lock!" ) );
         here.furn_set( target, furn_f_gunsafe_mj );
-    } else if( lock_roll > ( 1.5 * pick_roll ) ) {
+    } else if( lock_roll > ( 1.5 * pick_roll ) && !who.has_proficiency( proficiency_prof_lockpicking )  ) {
```

Comment:
So, this makes it completely impossible to damage your tools (no matter how difficult a lock) if you have proficiency_prof_lockpicking, not even prof_lockpicking_expert.

Given that the chance to damage is already skill-dependent, I think this needs to be *at least* checking for the lockpicking expert proficiency to say there's no chance to damage tools, ever.

Reply:
hmmm, you're right. 
but a survivor with lockpicking expert probably has a high enough devices skill not to run into this problem. so it might be a better idea to come up with an overhaul/upgrade to the lockpicking system instead of a balance change. I'll workshop some ideas on the forum.